src/com/yetanalytics/lrs_admin_ui/functions/time.cljs

```diff
@@ -2,28 +2,37 @@
   (:require [goog.string :refer [format]]
             [goog.string.format]))
 
-(def tz-offset-mins
-  (.getTimezoneOffset (js/Date.)))
-
-(def tz-offset-string
-  (format "%s%02d:%02d"
-          (if (pos-int? tz-offset-mins)
-            "-"
-            "+")
-          (quot tz-offset-mins 60)
-          (rem tz-offset-mins 60)))
+(defn tz-offset-mins*
+  ([]
+   (tz-offset-mins* (js/Date.)))
+  ([^js/Date d]
+   (.getTimezoneOffset d)))
+
+(defn tz-offset-string
+  [& {:keys [tz-offset-mins]}]
+  (let [offset (or tz-offset-mins
+                   (tz-offset-mins*))]
+    (format "%s%02d:%02d"
+            (if (pos-int? offset)
+              "-"
+              "+")
+            (quot offset 60)
+            (rem offset 60))))
 
 (defn local-datetime->utc
   [local-datetime-str]
   (-> local-datetime-str
-      (str tz-offset-string)
+      (str (tz-offset-string))
       (js/Date.)
       .toISOString))
 
 (defn utc->local-datetime
-  [utc-str]
+  [utc-str & {:keys [tz-offset-mins]}]
```

Comment:
I guess there's no need for the kwarg anymore if we're just forcing locale for all the tests?

Reply:
but yeah I'll remove for simplicity's sake

Reply:
Makes sense. But then:
1. Wouldn't we want to add a kwarg to `local-datetime->utc` to make thing symmetrical (in that case it would be passed to `tz-offset-string`)?
2. Would it be possible to add `:or (tz-offset-mins* (new js/Date utc-str))` to the kwarg map, instead of having the `or` in the function body? (I forgot if you can actually put function calls in there.)

Reply:
Ah nevermind lol